docs/nmt-lib.md

```diff
@@ -44,11 +44,13 @@ idSize := tree.NamespaceSize() // outputs 1
 
 ### Ignore Max Namespace
 
-If the NMT is configured with `IgnoreMaxNamespace` set to true, then the calculation of the namespace ID range of non-leaf nodes in the [namespace hash function](./spec/nmt.md#namespaced-hash) will change slightly.
-That is, when determining the upper limit of the namespace ID range for a tree node `n`, with children `l` and `r`, the maximum possible namespace ID, equivalent to `NamespaceIDSize()` bytes of `0xFF`, or `2^NamespaceIDSize()-1`,
-should be omitted if feasible (in the preceding code example with the ID size of `1` byte, the maximum possible namespace ID would be `0xFF`).
-This is achieved by taking the maximum value among the namespace IDs available in the range of node's left and right children (i.e., `n.maxNs = max(l.minNs, l.maxNs , r.minNs, r.maxNs))`, which is not equal to the maximum possible namespace ID value.
-If such a namespace ID does not exist, the `maxNs` is calculated as normal, i.e., `n.maxNs = max(l.maxNs , r.maxNs)`.
+If the NMT is configured with `IgnoreMaxNamespace` set to true (the flag is explained [here](#nmt-initialization-and-configuration)), then the calculation of the namespace ID range of non-leaf nodes in the [namespace hash function](./spec/nmt.md#namespaced-hash) will change slightly.
+That is, when determining the upper limit of the namespace ID range for a tree node, the maximum possible namespace `maxPossibleNamespace` should not be taken into account. 
+(In the preceding code example with the ID size of `1` byte, the value of `maxPossibleNamespace` is $2^8-1 = 0xFF$.)
+
+Concretely, for a node `n` with children `l` and `r`, the namespace ID is the largest namespace value from `l` and `r` different from  `maxPossibleNamespace`, if such a namespace ID exists.
+Otherwise, if all candidate values are equal to `maxPossibleNamespace`, the namespace ID of `n` is set to `maxPossibleNamespace`.
+Precisely, if a set `C = {ns \in {l.minNs, l.maxNs, r.minNs, r.maxNs}: ns < maxPossibleNamespace}` is not empty, `n.maxNs = max(C)`. If `C` is empty, `n.maxNs = maxPossibleNamespace`.
```

Comment:
The `\in` does not seem to be rendered properly in the output file. As a suggestion, it can be replaced with the word `in`.

Reply:
Thanks @staheri14 ! 
I pushed a commit in which I used latex rendering only for \in. I am not sure that it looks good, though (there is a difference in rendering between that and other symbols, e.g., `a `$\in$` A`.

As a second option, I wrapped everything in latex (please check how the rendered doc [here](https://github.com/ivan-gavran/nmt/blob/ivan/ignoreMaxNsExplanationLatex/docs/nmt-lib.md)). This looks much better when rendered but introduces perhaps unnecessary formalism in writing.

Which of the two options you find better?  (There is also the third option, which you originally suggested, to simply use `in`, though I'm afraid that might be a bit confusing.)

Reply:
The LaTeX form of the math symbol `\in` is more readable. As an alternative approach, we could keep the math symbols in LaTeX format while leaving the variables as normal text.

```
`C` $= {$ `ns` $\in {$`l.minNs`, `l.maxNs`, `r.minNs`, `r.maxNs` $}:$ `ns` $<$ `maxPossibleNamespace` $}$
```

Reply:
I like that - pushed a commit. (Also added different braces sizes.)